src/EditorFeatures/Test/Workspaces/ClassificationTypeNamesTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Reflection;
+using Microsoft.CodeAnalysis.Classification;
+using Microsoft.VisualStudio.Text.Classification;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces
+{
+    public class ClassificationTypeNamesTests
+    {
+        public static IEnumerable<object[]> AllClassificationTypeNames
+        {
+            get
+            {
+                foreach (var field in typeof(ClassificationTypeNames).GetFields(BindingFlags.Static | BindingFlags.Public))
+                {
+                    yield return new object[] { field.Name, field.GetRawConstantValue() };
+                }
+            }
+        }
+
+        [Theory]
+        [MemberData(nameof(AllClassificationTypeNames))]
+        [WorkItem(25716, "https://github.com/dotnet/roslyn/issues/25716")]
+        public void ClassificationTypeExported(string fieldName, object constantValue)
+        {
+            Assert.IsType<string>(constantValue);
```

Comment:
Random tip: Assert.IsType returns the type cast to the underlying type, so you can nicely combine this with the line below.